src/Raven.Server/Documents/Indexes/MapReduce/Static/ReduceMapResultsOfStaticIndex.cs

```diff
@@ -68,6 +70,8 @@ IEnumerator IEnumerable.GetEnumerator()
                 return GetEnumerator();
             }
 
+            public BlittableJsonReaderObject Current => _enumerator.Current?.BlittableJson;
```

Comment:
Isn't it disposed here?
https://github.com/ravendb/ravendb/blob/v5.3/src/Raven.Server/Documents/Indexes/MapReduce/ReduceMapResultsBase.cs#L301

Reply:
And here we try to access it:
https://github.com/ravendb/ravendb/blob/73ea69110cf9a6922abd9183c5e61ce0117bfa52/src/Raven.Server/Documents/Indexes/MapReduce/ReduceMapResultsBase.cs#L333

Reply:
Ouch. You're right. Great catch.

Reply:
Great catch

Reply:
I need to think more about how to fix is. Simply trying to move the disposal to `finally` is not sufficient because `AggregateBranchPage` might be called recursively in some cases.

Reply:
@grisha-kotler Handled in https://github.com/ravendb/ravendb/pull/13782/commits/04942800898b0beadaf98c1618b27b9d64d5915d. Please check it.